join-us.html

```diff
@@ -40,57 +40,32 @@
-                        skilled in the following one or more areas we would love to hear from you:<br></p>
-
-                    <ul class="mt-2">
-                        <li>Collaboration</li>
-                        <li>Leadership</li>
-                        <li>Digital Marketing</li>
-                        <li>Business Communication</li>
-                        <li>Programming</li>
+            <div class="row mx-auto py-md-5 align-items-center">
+                <!-- text div -->
+                <div class="col-md-6">
+                    <h1 style="font-weight: 700;"><span class="font-weight-bold">Join</span> the Team</h1>
+                    <p>
+                        We are always on the lookout for top talent to join our SEF Management Team. If you are skilled in the following one or more areas we would love to hear from you:
+                    </p>
+                    <ul class="list-unstyled text-left">
+                        <li> <i class="fa fa-award mr-2"></i></i>Collaboration</li>
+                        <li><i class="fa fa-award mr-2"></i></i>Leadership</li>
+                        <li><i class="fa fa-award mr-2"></i></i>Digital Marketing</li>
+                        <li><i class="fa fa-award mr-2"></i></i>Business Communication</li>
+                        <li><i class="fa fa-award mr-2"></i></i>Programming</li>
                     </ul>
+                    <a href="#faq-section">
+                        <button class="btn btn-default text-uppercase mt-2" href="examples">Learn more</button>
+                    </a>
                 </div>
-                <a href="#faq-section">
-                    <div class="btn btn-default">
-                        Learn more
-                    </div>
-                </a>
-            </div>
 
-            <!--            For mobile view-->
-            <div class="d-block d-md-none mt-5 py-4">
-                <div class="display-1 font-weight-bold my-auto d-flex text-default">Join<span
-                        class="font-weight-light">&nbsp;the Team</span></div>
-                <div class="px-0 text-justify">
-                    <p class="mt-auto mb-auto">We are always on the lookout for top talent to join our SEF Management Team. If you are
-                        skilled in the following one or more areas we would love to hear from you:<br></p>
-
-                    <ul class="mt-2">
-                        <li>Collaboration</li>
-                        <li>Leadership</li>
-                        <li>Digital Marketing</li>
-                        <li>Business Communication</li>
-                        <li>Programming</li>
-                    </ul>
+                <!-- image div -->
+                <div class="col-md-6 mt-5 mt-md-0">
+                    <img src="assets/img/join_us/heroNew.svg" class="img-fluid w-150">
```

Comment:
```suggestion
                    <img src="assets/img/join_us/hero.svg" class="img-fluid w-150">
```

The new image file is missing. Replace the old image with the new image. Commit that as well.